etc/debian/control

```diff
@@ -0,0 +1,36 @@
+Source: scancode-toolkit
+Section: misc
+Priority: optional
+Maintainer: Debian Python Modules Team <python-modules-team@lists.alioth.debian.org>
+Uploaders:Phillipe Ombredanne<pombredanne@nexb.com>, 
+	  Ayush Jain <aj4ayushjain@gmail.com>
+Build-Depends:  debhelper (>= 10), 
+		dh-python, 
+		python-all, 
+		python-setuptools,
+		python-dev,
+		xz-utils,
+		zlib1g,
+		libxml2-dev,
+		libxslt1-dev,
+		libbz2-1.0 | bzip2
+Standards-Version: 4.1.2
+Homepage: https://github.com/nexB/scancode-toolkit
+#Vcs-Git: https://anonscm.debian.org/git/python-modules/packages/scancode-toolkit.git
+#Vcs-Browser: https://anonscm.debian.org/cgit/python-modules/packages/scancode-toolkit.git/
+#Testsuite: autopkgtest-pkg-python
+
+Package: scancode-toolkit
+Architecture: all
+Depends: python-six,
+	 python-unicodescv,
+	 python-pycyptodome,
```

Comment:
pycyptodome dos not look like the correct name?

Reply:
right, another typo, the correct name is python-pycryptodrome

Reply:
i checked python-cryptodome is the correct name.